src/docfx/template/TemplateEngine.cs

```diff
@@ -174,15 +174,30 @@ public void CopyAssetsToOutput(Output output, bool selfContained = true)
         }
 
         // content for *.mta.json
-        var metadata = new JObject(templateMetadata.Properties().Where(p => !p.Name.StartsWith("_")))
+        var metadata = new JObject(templateMetadata.Properties().Where(p => !p.Name.StartsWith("_") && p.Name != "xrefs"))
```

Comment:
Why do we need to change this file. Suppose we are just adding a new metadata, that new metadata should flow through the system without touching the template engine?